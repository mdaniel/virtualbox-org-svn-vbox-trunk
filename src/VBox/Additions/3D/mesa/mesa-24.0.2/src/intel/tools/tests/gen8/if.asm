(+f0.0) if(8)   JIP: LABEL0       UIP: LABEL1                   { align1 1Q };
(-f0.0) if(8)   JIP: LABEL0       UIP: LABEL1                   { align1 1Q };
LABEL0:
(-f0.0) if(16)  JIP: LABEL1       UIP: LABEL1                 { align1 1H };
(+f0.0) if(16)  JIP: LABEL1       UIP: LABEL1                 { align1 1H };
(+f0.0) if(32)  JIP: LABEL1       UIP: LABEL1                 { align1 };
LABEL1:
