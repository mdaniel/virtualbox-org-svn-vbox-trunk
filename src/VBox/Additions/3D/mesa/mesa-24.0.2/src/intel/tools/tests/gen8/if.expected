22 00 61 00 20 0e 00 20 50 00 00 00 20 00 00 00
22 00 71 00 20 0e 00 20 40 00 00 00 10 00 00 00
22 00 91 00 20 0e 00 20 30 00 00 00 30 00 00 00
22 00 81 00 20 0e 00 20 20 00 00 00 20 00 00 00
22 00 a1 00 20 0e 00 20 10 00 00 00 10 00 00 00
